### ISSUE TITLE
Question about Equation 17 in the paper

### ISSUE THREAD

**Issue by chuong98** [adityac94/Grad_CAM_plus_plus#3]
Hi, it seems to me that you can simplify the equation Equ 17 by dividing both numerator and denominator by (dS/dA)^2, to get : 

$\alpha$

**Comment by adityac94** [comment]
You can't divide by infinitesimals. That does not make sense

**Comment by chuong98** [comment]
No, taking the limit when dS/dA converges to 0, alpha converges to 0. Otherwise, you can divide it.
